tests/tests.py

```diff
@@ -44,16 +44,7 @@ def assert_tags_equal(self, qs, tags, sort=True, attr="name"):
         self.assertEqual(got, tags)
 
     def _get_form_str(self, form_str):
-        if django.VERSION >= (1, 3):
-            form_str %= {
-                "help_start": '<span class="helptext">',
-                "help_stop": "</span>"
-            }
-        else:
-            form_str %= {
-                "help_start": "",
-                "help_stop": ""
-            }
+        form_str %= {"help_start": "", "help_stop": ""}
```

Comment:
This is incorrect. This should be the *first* branch of the conditional (`if django.VERSION >= (1, 3):`).